src/Forte.Optimizely.ContentUsage/Api/Services/ContentTypeService.cs

```diff
@@ -2,15 +2,22 @@
 using System.Collections.Generic;
 using System.Linq;
 using EPiServer.DataAbstraction;
+using EPiServer.ServiceLocation;
+using Forte.Optimizely.ContentUsage.Api.Features.ContentType;
 
 namespace Forte.Optimizely.ContentUsage.Api.Services;
 
 public class ContentTypeService
 {
     private readonly IContentTypeRepository _contentTypeRepository;
+    private readonly ServiceAccessor<ContentTypeUsagesDB> _contentTypeUsagesDBAccessor;
```

Comment:
once ContentTypeUsagesDB is registered by services.AddSingleton we can get rid of this ServiceAccessor thing